### ISSUE TITLE
Some issues on MyOwn data

### ISSUE THREAD

**Issue by iamweiweishi** [zudi-lin/rcan-it#14]
Hi, 
I have collected about 20000 images. And the HR-resolution is around 600*600. 
When the training starts, everything looks fine.
After about 10000 iterations, the loss become larger and larger. Finally, the loss become nan.
Any suggestions on this kind of issue?

Best regards,
Thank you.

**Comment by zudi-lin** [comment]
hello @iamweiweishi, we described in our paper that the training is sensitive to the learning rate (see 4.1 in https://arxiv.org/pdf/2201.11279.pdf). I would suggest you decrease the learning rate by 8x and try the training again.
